pkg/nemesis/pod_kill.go

```diff
@@ -38,8 +38,9 @@ func podKillNodes(nodes []cluster.Node, n int) []*core.NemesisOperation {
 	for i := 0; i < n; i++ {
 		ops[indices[i]] = &core.NemesisOperation{
 			Type:        core.PodKill,
-			InvokeArgs:  []interface{}{nodes[i], freq},
-			RecoverArgs: []interface{}{nodes[i]},
+			Node:        &nodes[i],
```

Comment:
For kill, and netem chaos, I record target node here, this change aims to simplify the nemesis apply phase.